documents.go

```diff
@@ -47,3 +60,68 @@ func (cl *DocumentsClientGetAll) Parent(id DocumentID) *DocumentsClientGetAll {
 func (cl *DocumentsClientGetAll) Do(ctx context.Context, fn func(*Document, error) bool) error {
 	return nil
 }
+
+// Create returns a client for creating a single document in the specified collection.
+// collectionID: The ID of the collection to which the document will be added.
+// title: The title of the document (optional, set to empty string to omit).
+// text: The content text of the document.
+// publish: A boolean indicating whether the document should be published.
+func (cl *DocumentsClient) Create(collectionID CollectionID, title string, text string, publish bool, parentDocumentId ParentDocumentID, templateId TemplateID, template bool) *DocumentsClientCreate {
+	return newDocumentsClientCreate(cl.sl, collectionID, title, text, publish, parentDocumentId, templateId, template)
+}
+
+// DocumentsClientCreate is a client for creating a single document.
+type DocumentsClientCreate struct {
+	sl *sling.Sling
+}
+
+// newDocumentsClientCreate creates a new DocumentsClientCreate instance to create a single document in the specified collection.
+// sl: The Sling client used to make HTTP requests.
+// collectionID: The ID of the collection to which the document will be added.
+// title: The title of the document.
+// text: The content text of the document (optional, set to empty string to omit).
+// publish: A boolean indicating whether the document should be published (optional, set to false to omit).
+// parentDocumentId: The ID of the parent document if the document is being created as a child (optional, set to empty string to omit).
+// templateId: The ID of the template to use for creating the document (optional, set to empty string to omit).
+// template: A boolean indicating whether the document is a template (optional, set to false to omit).
+// https://www.getoutline.com/developers#tag/Documents/paths/~1documents.create/post
+func newDocumentsClientCreate(sl *sling.Sling, collectionID CollectionID, title string, text string, publish bool, parentDocumentId ParentDocumentID, templateId TemplateID, template bool) *DocumentsClientCreate {
+	data := struct {
+		CollectionID     CollectionID     `json:"collectionId"`
+		Title            string           `json:"title"`
+		Publish          bool             `json:"publish,omitempty"`
+		Text             string           `json:"text,omitempty"`
+		ParentDocumentId ParentDocumentID `json:"parentDocumentId,omitempty"`
+		TemplateID       TemplateID       `json:"templateId,omitempty"`
+		Template         bool             `json:"template,omitempty"`
+	}{
+		CollectionID:     collectionID,
+		Title:            title,
+		Publish:          publish,
+		Text:             text,
+		ParentDocumentId: parentDocumentId,
+		TemplateID:       templateId,
+		Template:         template,
+	}
+
+	copy := sl.New()
+	copy.Post(common.DocumentsCreateEndpoint()).BodyJSON(&data)
+	return &DocumentsClientCreate{sl: copy}
+}
+
+// Do creates a document with the specified title and content.
```

Comment:
Which specified `title` and `content`? 😉 
`Do` just makes the request for the given `DocumentsClientCreate` struct 😁

Reply:
Ups, probably copy paste :dancers: I'll also remove the extensive documentation :+1: